server/src/main/scala/services/graphstages/ArrivalsGraphStage.scala

```diff
@@ -257,11 +259,18 @@ class ArrivalsGraphStage(name: String = "",
       mergeBestFieldsFromSources(baseArrival, mergeArrival(key).getOrElse(baseArrival))
     }
 
+
     def mergeArrival(key: UniqueArrival): Option[Arrival] = {
-      val maybeBestArrival: Option[Arrival] = (liveArrivals.get(key), liveBaseArrivals.get(key)) match {
+      val maybeBestArrival: Option[Arrival] = (
+        liveArrivals.get(key),
+        liveBaseArrivals.get(key).map(arrivalDataSanitiser.withSaneEstimates)) match {
```

Comment:
probably extract `liveBaseArrivals.get(key).map(arrivalDataSanitiser.withSaneEstimates)` to a val to give it a descriptive name and save the match being split onto multiple lines

Reply:
done.